redisinsight/ui/src/packages/ri-explain/src/Explain.tsx

```diff
@@ -0,0 +1,254 @@
+import React, { useEffect, useState, useRef } from 'react'
+import { Model, Graph } from '@antv/x6'
+import { register} from '@antv/x6-react-shape'
+import Hierarchy from '@antv/hierarchy'
+
+import {
+  CoreType,
+  EntityInfo,
+  ParseExplain,
+  ParseGraph,
+  ParseProfile,
+  ParseProfileCluster,
+} from './parser'
+import { ExplainNode, ProfileNode } from './Node'
+
+interface IExplain {
+  command: string
+  data: [{response: string[] | string | any}]
+}
+
+export default function Explain(props: IExplain): JSX.Element {
+  const command = props.command.split(' ')[0].toLowerCase()
+
+  if (command.startsWith('graph')) {
+    const info  = props.data[0].response
+    const resp = ParseGraph(info);
+    return (
+      <ExplainDraw
+        data={resp}
+        type={command.endsWith('explain') ? CoreType.Explain : CoreType.Profile}
+      />
+    )
+  }
+
+  if (command == 'ft.profile') {
+    const info = props.data[0].response[1]
+
+    let data: EntityInfo;
+    let profilingTime: IProfilingTime = {};
+
+    if (info.length > 5 && typeof info[0] === 'string' && info[0].toLowerCase().startsWith('shard')) {
+      let cluster: Object;
+      [cluster, data] = ParseProfileCluster(info)
```

Comment:
remove pls `let cluster: Object;`
enough `let [cluster] = ParseProfileCluster(info)` you don't use `data` bellow in this statement